lib/schema/map.js

```diff
@@ -12,23 +12,48 @@ const MongooseError = require('../error/mongooseError');
 const Schema = require('../schema');
 const utils = require('../utils');
 
-/*!
- * ignore
- */
-
 class SchemaMap extends SchemaType {
-  constructor(key, options, _schemaOptions, parentSchema) {
+  /**
+   * Map SchemaType constructor.
+   *
+   * @param {String} path
+   * @param {Object} options
+   * @param {Object} schemaOptions
+   * @param {Schema} parentSchema
+   * @inherits SchemaType
+   * @api public
+   */
+
+  constructor(key, options, schemaOptions, parentSchema) {
     super(key, options, 'Map', parentSchema);
     this.$isSchemaMap = true;
-    
     // Create the nested schema type for the map values
-    this._createNestedSchemaType(parentSchema, key, options, _schemaOptions);
+    this._createNestedSchemaType(parentSchema, key, options, schemaOptions);
   }
 
+  /**
+   * Sets a default option for all Map instances.
+   *
+   * @param {String} option The option you'd like to set the value for
+   * @param {Any} value value for option
+   * @return {undefined}
+   * @function set
+   * @static
```

Comment:
From what i can tell, `set` is not marked `static`

Reply:
Good catch, I fixed this in the 8.20 branch